Evaluation/Input/Content/Concentration_time_profiles.md

```
@@ -0,0 +1 @@
+Simulated versus observed concentration-time profiles of all data listed in [Section 2.2.2](#222-clinical-data) are presented below.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)